Tasks/JenkinsQueueJobV2/job.ts

```diff
@@ -433,7 +452,9 @@ export class Job {
             }
         }).auth(thisJob.queue.TaskOptions.username, thisJob.queue.TaskOptions.password, true)
         .on('error', (err) => {
-            throw err;
+            if (thisJob.retriesLeft < 1) {
```

Comment:
Why do we need this condition here? Should the task print error anyway - even if there are some retry attempts remaining?

Reply:
Because that would fail the task, and we try to avoid it

Reply:
Although we can print error in this construction

Reply:
For this case - should we trigger retry here instead and print the error?

Reply:
Yeah, i've implemented message there, but i dont think we should move it there, since it wont trigger on status code errors then, It would be better to leave placement as it is I think.